it/controllers/OtherReasonControllerISpec.scala

```diff
@@ -516,13 +516,13 @@ class OtherReasonControllerISpec extends IntegrationSpecCommonBase {
         (SessionKeys.dateCommunicationSent, "2020-02-08T12:00:00"),
         (SessionKeys.journeyId, "1234")
       )
-      val request = await(controller.onPageLoadForUploadComplete(NormalMode)(fakeRequestWithCorrectKeys))
+      val request = await(controller.onPageLoadUploadList(NormalMode)(fakeRequestWithCorrectKeys))
```

Comment:
Could you change the "GET /upload-complete" to the correct URL:
https://github.com/hmrc/penalties-appeals-frontend/pull/185/files#diff-1f25331e29c9f185c275d716035c984e6f80e928d6470edf5c800ab02f647228R508